### PR TITLE
Billing manager new feature add/edit MBO form.

### PR DESCRIPTION
**Pull request by sjpadgett**
style changes to better support institutional billing.
dlgopen getting closer to removing include_opener.php.
Move ub04 form edit to dialog.

below added by bradymiller:
Up For Grabs demo for this PR is here: https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller @stephenwaite It just seems reasonable for the MBO to be in claims. This version will allow user to add form if not exist or edit one that does. I didn't add a turn on/off to globals and am looking for implementation suggestions. Added style changes for more screen space and toolbar behavior.
An up for grab(w/data) would be nice so we can test. Hopefully Stephen you could give me a hand on that.

**Comment by bradymiller** [comment]
hi,
Set up an Up For Grabs demo (with demo data) here:
https://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

**Comment by stephenwaite** [comment]
really cool @sjpadgett , testing fine for me.  Nice work.

**Comment by sjpadgett** [comment]
@bradymiller I've tested this in tabs and frames on linux and windows and see no issues. Appears good to go. Do you have any comments?

File: interface/forms/misc_billing_options/new.php [hunk 1]
```diff
@@ -20,15 +22,23 @@
...
     die(xlt("Internal error: we do not seem to be in an encounter!"));
 }
-//only one misc billing form so grab if exists
+//only one misc billing form per encounter so grab if exists
 $formid = 0 + (isset($_GET['id']) ? $_GET['id'] : 0);
 if (empty($formid)) {
     $mboquery = sqlquery("SELECT `fmbo`.`id` FROM `form_misc_billing_options` AS `fmbo`
                           INNER JOIN `forms` ON (`fmbo`.`id` = `forms`.`form_id`) WHERE
-                          `forms`.`deleted` = 0 AND `forms`.`form_name` = 'Misc Billing Options' AND
+                          `forms`.`deleted` = 0 AND `forms`.`formdir` = 'misc_billing_options' AND
```
Comment (bradymiller) [inline interface/forms/misc_billing_options/new.php]: very smart change :)

**Comment by bradymiller** [comment]
hi @sjpadgett ,
Code looks good to me (sadly, I was unable to find any escaping or translation issues :) ). Rec. bringing it into the codebase and thanks for yet another very nice contribution! -brady

**Comment by bradymiller** [comment]
btw, rec bringing this in as a 'Squash and merge'; then all the library/dialog.js changes that are already in the codebase will magically disappear from the commit.

File: interface/forms/misc_billing_options/new.php (hunk 1 above)
Comment (sjpadgett) [reply interface/forms/misc_billing_options/new.php]: I do still have fleeting moments of clarity.

Comment (bradymiller) [reply interface/forms/misc_billing_options/new.php]: yep, I remember having those in the past. It's been a couple years for me :)

**Comment by sjpadgett** [comment]
You taught me well and you're welcome. I hope to back test this on 5.0.1 and maybe this could be a patch. I'll let ya know.

**Pull request merged**